src/hugo/content/en-us/implementations/aws/software_update/IoT_Jobs/_index.md

```diff
@@ -0,0 +1,156 @@
+3. An _Application_ or user creates a _Job_ in IoT Core to be executed on the device. Data needed for the _Job_ is stored in an S3 bucket.
+4. The IoT Jobs service publishes a message to the jobs topic `$aws/things/thingName/jobs/notify-next` notifying the _Device_ of a pending _Job_.
+5. The _Device_ receives the notification and retrieves the job details from the S3 bucket. 
+6. The _Device_ can update the status of a job execution by publishing status messages to the job update topic `$aws/things/thingName/jobs/JobID/update`. These statuses can include _"IN_PROGRESS"_, _"FAILED"_, _"SUCCEEDED"_ or _"REJECTED"_.
+7. (Optional) Management and monitoring applications can subscribe to certain topics to keep track of the status of jobs. 
+
+## Assumptions
+This implementation approach assumes devices are already provisioned within AWS IoT. Please refer to the [Getting started with AWS IoT Core](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html) for details on connecting your device to AWS IoT. 
+This implementation also assumes that proper permissions have already been granted within IoT Core Policies and S3 resources. 
+
+## Implementation
+
+To create a job, you must first create a job document that is a description of the remote operations to be performed by the devices. A job document contains one or more URLs where the device can download an update or some other data. The job document can be stored in an Amazon S3 bucket, or be included inline with the command that creates the job.
+
+__Sample Job Document__:
+{{< tabs groupId="device-code">}}
+{{% tab name="example-job.json" %}}
+
+```json
+{ 
+"operation": "download",
+"version" : "1.0", 
+"directory" : "/installs",
+"files":{
+    "fileName" : "sample_file",
+    "url" : "S3_URL or S3_Presigned_URL"
+        }
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% notice note %}}
+Your job document can contain a presigned Amazon S3 URL that points to your code file (or other file). Presigned Amazon S3 URLs are valid for a limited amount of time and so are not generated until a device requests a job document. Because the presigned URL has not been created when you are creating the job document, you put a placeholder URL in your job document instead. A placeholder URL looks like the following: **${aws:iot:s3-presigned-url:https://s3.region.amazonaws.com/<_bucket_>/<_codefile_>}** where bucket is the Amazon S3 bucket that contains the code file and code file is the Amazon S3 key of the code file
+{{% /notice %}}
+
+{{% notice note %}}
+For more job document examples, see the [jobs-agent.js](https://www.npmjs.com/package/aws-iot-device-sdk) example in the AWS IoT SDK for JavaScript.
+{{% /notice %}}
+
+{{< tabs groupId="device-code">}}
+{{% tab name="python" %}}
+
+Please refer to this [jobs sample](https://github.com/aws/aws-iot-device-sdk-python-v2/blob/main/samples/jobs.py) for more details and examples.
+
+- Install SDK from PyPI: `python3 -m pip install AWSIoTPythonSDK`
```

Comment:
The v1 device SDKs have been deprecated. This code can be left in but I'll open an issue to have it refactored using the v2 device SDK.